ambry-api/src/main/java/com/github/ambry/config/QuotaConfig.java

```diff
@@ -73,11 +67,19 @@
   public String quotaManagerFactory;
 
   /**
-   * The mode in which quota throttling is being done (TRACKING/THROTTLING).
+   * The mode in which quota throttling is being done (TRACKING/THROTTLING). To throttle the requests, you have to change
+   * the mode to THROTTLING and turn on requestThrottlingEnabled or storageQuotaConfig.shouldThrottle (or both).
    */
   @Config(THROTTLING_MODE)
   public QuotaMode throttlingMode;
 
+  /**
+   * Config to enable request throttling on customer's account or container.
+   */
+  @Config(REQUEST_THROTTLING_ENABLED)
+  @Default("true")
```

Comment:
Why is request throttling true by default. I think it should be false by default, and we can make it true where we need it to be true.

Reply:
The idea is that Mode controls whether we want to throttle or not, so by default, when we add an enforcer to the list, we are expecting it follows the global settings. So if we don't want it follow the setting, we will set it on purpose. Thus the default value should be "true".

Reply:
Just wanted to check that this means that we now have separate config to enable/disable for each quota type (`quotaConfig.requestThrottlingEnabled` for read/write quota and `quotaConfig.storageQuotaConfig.shouldThrottle` for storage quota)and there is no global config to enable/disable all quotas.  

Just a question, would it be beneficial to also have a global config to enable/disable all quotas (with default value as true)?

Reply:
@Arun-LinkedIn We do have a global configuration, but not to turn off the quota enforcers, but to change the mode of quota enforcers. Currently there are two modes in quota enforcers, Tracking and Throttling. We have a global configuration to pick either mode for all quota enforcers. The default mode is Tracking, which means we are not going to throttle any request.